oat-sa/extension-tao-itemqti
# feat: make clickable every word in hottext interaction

Ticket: https://oat-sa.atlassian.net/browse/AUT-3752

## What's Changed

- Introduced new button which make every separated by whitespace part of selection as clickable response option
- Selection of every word as clickable response option can be canceled separately
- A new button is not visible for a selection that does not contain spaces

## How to test
- Create a test item with the hottext interaction.
- Highlight the part of it (or full text).
- Click the right hand button.
- Check every highlighted word separated by space is created as a separate clickable choice.
- Try to use different scenarios.

**Deployed** to: http://test-viktar.playground.kitchen.it.taocloud.org:41531/tao/Main/login
